static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -2,24 +2,12 @@
 
 import config from 'lib/config';
 import memoize from 'lodash/memoize';
-import {
-    allRunnableTests,
-    firstRunnableTest,
-} from 'common/modules/experiments/ab-core';
-import {
-    runnableTestsToParticipations,
-    testExclusionsWhoseSwitchExists,
-} from 'common/modules/experiments/ab-utils';
+import { runnableTestsToParticipations } from 'common/modules/experiments/ab-utils';
 import {
     registerCompleteEvents,
     registerImpressionEvents,
     trackABTests,
 } from 'common/modules/experiments/ab-ophan';
-import {
-    getParticipationsFromLocalStorage,
-    setParticipationsInLocalStorage,
-} from 'common/modules/experiments/ab-local-storage';
```

Comment:
Is there no way to keep a user in a test group across pageviews?

Reply:
On CODE/PROD you can do this using the `opt/in` and `opt/out` endpoints: https://github.com/guardian/frontend/blob/master/applications/conf/routes#L61.

But for local development, the only way here would be by modifying your MVT ID. See https://github.com/guardian/ab-testing/#mvtid-calculator for a bit more on how to calculate this.

Reply:
I should add, I'm not at all against local storage for this or anything. But we should build this into the `ab-testing` library if we want it.

Reply:
Thanks for clarifying.

I’ve used the MVT ID calculator before and it’s very convenient! Better than going down the route of changing your request headers, which I’ve also done…